### DISCUSSION TITLE
Timeout in Django when large CSV download.  Could this be alleviated by async await in FASTAPI?

### DISCUSSION THREAD

**Discussion by scheung38** [fastapi/fastapi#3234] [Questions] [answered]
Could someone comment if timeout in Django due to large data and slow bandwidth context be alleviated by async await in FASTAPI?

**Comment by acidjunk** [comment, marked as answer]
No: but FastAPI has a simple background job execution mechanism.

 -> FWIW: the problem in the Django app probably isn't the download but generating the csv: When download has finally started the 30 sec. timeout isn't hit.
